text/0025-core-compute-resources-as-plugins.md

```diff
@@ -0,0 +1,77 @@
+- Feature Name: Core Compute Resources as Plugins
+- Start Date: 2016-10-21
+- RFC PR: (leave this empty)
+- Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Core Compute Resources should move to plugins
+
+# Motivation
+[motivation]: #motivation
+
+Core has been set up to allow compute resources as plugins for some while, and
+we already have several examples of this in use (Xen, Azure, DigitalOcean,
+etc). This has brought benefits:
+
+* able to release independatly of Core
+* make use of the modular Fog gems that are now available
+* separate logging of issues
+* dedicated maintainers who know the compute resource well
+
+In addition, it allows the core project to mark a particular resource as
+unmaintained when necessary (as happened with Xen). 
```

Comment:
To your first point - yes, but where would you make that mark? For plugins, we have http://projects.theforeman.org/projects/foreman/wiki/List_of_Plugins where you can clearly see which are unmaintained (and even easier if #30 goes ahead).

To your second point - I think you have a typo. I parse that as you agreeing with me, this is a reason to move the code to a plugin (gem) :)